pyscf/adc/radc.py

```diff
@@ -587,6 +515,38 @@ def density_matrix(myadc, T=None):
     return dm
 
 
+def analyze(myadc):
+
+    logger.info(myadc, "\n*************************************************************")
+    logger.info(myadc, "                Eigenvector analysis summary")
+    logger.info(myadc, "*************************************************************")
+
+    myadc.analyze_eigenvector()
+ 
+    if myadc.compute_properties == True:
+
+        logger.info(myadc, "\n*************************************************************")
+        logger.info(myadc, "            Spectroscopic factors analysis summary")
+        logger.info(myadc, "*************************************************************")
+
+        myadc.analyze_spec_factor()
+
+
+def compute_dyson_orb(myadc):
+     
+    X = myadc.X
+
+    if X is None:
+        U = np.array(myadc.U)
```

Comment:
Isn't U already an array here?

Reply:
No, myadc.U is the U obtained directly from the eigensolver.